### ISSUE TITLE
Videomaker: Should grid block patterns be Query Loops?

### ISSUE THREAD

**Issue by beafialho** [Automattic/themes#4892]
While setting up the [demo site](https://videomakerdemo.wordpress.com), I noticed that the grid block patterns aren't query loops. If they were, they'd only show post featured images, because having featured videos — as mentioned in [this](https://github.com/WordPress/gutenberg/issues/27617#issuecomment-918205913) Gutenberg issue — is not possible yet.

<img width="1093" alt="Captura de ecrã 2021-10-21, às 16 07 03" src="https://user-images.githubusercontent.com/65220155/138305942-0715ff76-c7a4-45ac-8189-714490f47eaa.png">

On top of that, if we add featured images to the posts, they will appear in the post pages, instead of showing the videos, which is the ultimate intent of this theme.

| Current Behavior  | Intended Behavior |
| ----------- | ----------- |
| <img width="987" alt="Captura de ecrã 2021-10-21, às 16 14 19" src="https://user-images.githubusercontent.com/65220155/138307097-c4ee4553-f133-4f02-8729-e3d820c7fd53.png">  | <img width="989" alt="Captura de ecrã 2021-10-21, às 16 14 28" src="https://user-images.githubusercontent.com/65220155/138307148-0eec4bc3-e8da-4395-b98e-2866f6c01d3f.png">  |

So, I wanted to ask what you think is the best approach here. Should the grid patterns be query loops? Or should the videos have manual links to the posts?

**Comment by MaggieCabrera** [comment]
mmh, I remember solving this for Spearhead, there may be a way to hack a "featured video" for the featured image block but we'd have to evaluate how that would transfer if/when the GB issue is resolved. Maybe if the "hack" is good enough we wouldn't change anything even then. I'm going to try and think about this.

**Comment by kjellr** [comment]
Yeah, I'm torn about it. We could hack something in here, but really we should hold off until there's a solution to https://github.com/WordPress/gutenberg/issues/27617#issuecomment-918205913. In the days of classic themes, we could've probably managed it, but I think we'll run into some weird issues for block themes (for example, even if we can hack it into the front end, what would the editor show? 😩). 

I think the easiest route for now might be to make each of those grid items videos, and then link to individual pages. For the pages themselves, we could consider using a "No featured image" page template by default, so that even if the user put a featured image in there, it wouldn't show up above the video.

**Comment by beafialho** [comment]
I get your point with that solution @kjellr.

However, if I were a user and I was using this theme because it's a video theme, I'd expect it to be easy to upload a featured video, instead of having to add it manually, just as it is when you upload a featured image and get your latest posts automatically. 

Is waiting and launching the theme only when we have that functionality an option? Or getting that functionality sooner?

**Comment by kjellr** [comment]
We chatted about this elsewhere, but I realized we never migrated this recommendation over here.

We shouldn't treat this like a blocker at all, since we can adjust the design to make it work today. I recommend that we redesign the content so that these seem less like posts and more like a grid of individual videos.

I think the dates are the element that _really_ make these seem like a grid of posts. Maybe we can change the dates to a short description instead? For example: 

<img width="882" alt="Screen Shot 2021-11-03 at 7 45 38 AM" src="https://user-images.githubusercontent.com/1202812/141473475-ce01538d-120d-4cd4-a2d9-d075ceb601be.png">

If that still seems too much like a post, maybe something like this is better? 

![Grid with Footer](https://user-images.githubusercontent.com/1202812/141474118-f3d282b8-9dc7-4d6a-af97-da309bc2ecf8.jpg)
